pkgs/applications/audio/flac2all/default.nix

```diff
@@ -0,0 +1,37 @@
+{ python3Packages, lib, flac, lame, opusTools, vorbis-tools, ffmpeg }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "flac2all";
+  version = "5.1";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "OBjlr7cbSx2WOIfZUNwHy5Hpb2Fmh3vmZdc70JiWsiI=";
+  };
+
+  # Not sure why this is needed, but setup.py expects this to be set
+  prePatch = ''
+    echo ${version} > ./flac2all_pkg/version
+  '';
+
+  propagatedBuildInputs = [
+    # Hard requirements
+    python3Packages.pyzmq
+    flac
+    lame
+    # Optional deps
+    opusTools
+    vorbis-tools
+    ffmpeg
+  ];
+
+  # Untested
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Multi process, clustered, FLAC to multi codec audio converter with tagging support";
+    homepage = "https://github.com/ZivaVatra/flac2all";
+    license = licenses.gpl3;
+    platforms = platforms.linux;
```

Comment:
Why not unix? which is the default and can be ommited.

Reply:
I've only tested on Linux, and it should be backwards compat to expand this if someone tests with non-Linux.

Reply:
If ofborg determines that it builds on darwin than this is good enough for now-